README.md

```diff
@@ -147,49 +147,34 @@ Rev 2 PCBs come with ARM 10-pin SWD headers and can be programmed via ST-Link. A
 ### ST-Link Software Installation
 
 ### Ubuntu
-1.  Download and build the ST-Link utilities:
-    ```
-    git clone https://github.com/stlink-org/stlink
-    cd stlink
-    cmake .
-    make
-    ```
-
-2. Copy the built binaries into their place:
-    ```
-    cd bin
-    sudo cp st-* /usr/local/bin
-    cd ../lib
-    sudo cp *.so* /lib32
-    ```
 
-3. Copy the udev rules to their place (this lets you run the st-link commands without using `sudo`):
-    ```
-    cd ../config/udev/rules.d
-    sudo cp 49-stlinkv* /etc/udev/rules.d/
-    ```
+1. sudo apt-install stlink-tools
+2. sudo apt-install stlink-gui
+3. sudo apt-install openocd
 
 ### Windows
 
 1. Download [st-link utility](http://www.st.com/content/st_com/en/products/development-tools/software-development-tools/stm32-software-development-tools/stm32-programmers/stsw-link004.html)
 
 ### Steps to Flashing a Rover Board
+**Important**: Ensure your PCB is powered prior to connecting the st-link to your computer (this is an ST bug)
 
 #### Ubuntu
 
-1. Cd into the folder containing the binary of the program to flash (eg. `MarsRover2020-firmware/build-<target name>/apps/<app name>`)
-
-2. Ensure that the ST-Link is connected to your computer and to the rover board.
-    - **Tip:** You can run `st-info --probe` to check if the ST-Link connection is detected
+Option 1: Use GUI
+1. Launch the GUI with `stlink-gui`
+2. Click connect
+3. Click Open and select your .bin file
+4. Click Flash (leave memory address as 0x8000000)
 
-3. Flash the program binary: `st-flash write <app name>.bin 0x8000000` (may need to run this command twice to get a successful flash)
-    - If the flashing was successful, you should see a blinking LED pattern on the board and the following message: `Flash written and verified! jolly good!`
+Option2: Use CLI
```

Comment:
Add a space: "Option 2"

Reply:
You can actually use github code suggestions to make a suggestion that he can just click accept on